create_deb.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+
+version=$(cat setup.py | sed -rn "s/.*version = '(.*?)'.*/\1/p")
```

Comment:
The version in setup.py is not being updated. Let's use the same version as is compiled into the binary.
```suggestion
version=$(echo GITVERSION | cpp -P -include gitversion.h | sed 's/"//g')
```